cmd/build/helmify/replacements.go

```diff
@@ -173,6 +173,7 @@ var replacements = map[string]string{
     - services/proxy
     - nodes/proxy
     - services/status
+    scope: 'Namespaced'
```

Comment:
Let's not modify existing behavior. Can you use helm to conditionally set the field separetly?